### DISCUSSION TITLE
Explorer crashes on startup with version 22H2 (OS Build 23466.1001)

### DISCUSSION THREAD

**Discussion by thanukrishnan** [valinet/ExplorerPatcher#1488] [Q&A] [answered]
Hi.

After the update to the latest preview release from Microsoft, the windows explorer crashes on startup. It got fixed only after I uninstalled explorerpatcher. I am aware that this version hasn't been tested/supported yet but would like to know if anybody else had the same issue.

**Comment by dungcoiads** [comment]
That's right, after updating the KB5026446 Explorer Patcher on my computer, I got an error, couldn't open Window Explorer.

**Comment by pyrates999** [comment]
Preview updates aren't supported by explorer patcher.  Uninstall all preview updates.

https://github.com/valinet/ExplorerPatcher/wiki/Symbols

**Comment by thanukrishnan** [comment, marked as answer]
Found a fix,
1. Uninstall the app
2. Delete the regkey "Computer\HKEY_CURRENT_USER\Software\ExplorerPatcher"
3. Download and install the latest version from Git release
This worked for me

**Reply by dungcoiads** [reply]
Thanks, it worked

**Comment by freeman-hu** [comment]
Yes, after installing KB5026446, I only got a black screen with a functioning cursor.

It was interesting that when I logged in not as a user with administrative privileges, but with the actual,  real Administrator account, there was no issue. Removing PatchExplorer + Reboot solved the problem. I hope there will be an update for it soon.  (HU->EN translated by ChatGPT, sorry)
